tasks/main.yml

```diff
@@ -0,0 +1,87 @@
+---
+# tasks file for atb-ansible-ghostagent
+- name: Download microsoft repository
+  become: true
+  ansible.builtin.shell:
+    cmd: "wget https://packages.microsoft.com/config/{{ ansible_distribution | lower }}/{{ ansible_distribution_version }}/packages-microsoft-prod.deb -O /opt/packages-microsoft-prod.deb"
```

Comment:
you could also use the get_url-module: https://docs.ansible.com/ansible/latest/collections/ansible/builtin/get_url_module.html